<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Wocson</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>581</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>202</red>
        <green>202</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>202</red>
        <green>202</green>
        <blue>202</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>184</red>
        <green>194</green>
        <blue>179</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>BR</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>0</y>
      <width>775</width>
      <height>486</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_cockpit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>&amp;Control</string>
     </attribute>
     <widget class="QPushButton" name="btnRunC">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>50</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>21</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>220</green>
            <blue>168</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>220</green>
            <blue>168</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>220</green>
            <blue>168</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>Engine</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="KLed" name="kledRunC">
      <property name="geometry">
       <rect>
        <x>52</x>
        <y>0</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="state">
       <enum>KLed::Off</enum>
      </property>
      <property name="look">
       <enum>KLed::Sunken</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_archive">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>&amp;Archive</string>
     </attribute>
     <widget class="QPushButton" name="btnRunA">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>40</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="KLed" name="kledRunA">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>1</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>21</height>
       </size>
      </property>
      <property name="state">
       <enum>KLed::Off</enum>
      </property>
      <property name="look">
       <enum>KLed::Sunken</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_wsettings">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="text" stdset="0">
      <string>Settings</string>
     </property>
     <attribute name="title">
      <string>&amp;Settings</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>0</y>
        <width>748</width>
        <height>442</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox01">
         <property name="minimumSize">
          <size>
           <width>356</width>
           <height>276</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>356</width>
           <height>276</height>
          </size>
         </property>
         <property name="title">
          <string comment="aaaaaaaaaaa" extracomment="cccccccccccc">Base settings</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>25</y>
            <width>29</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Level:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>19</x>
            <y>45</y>
            <width>19</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Dip:</string>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>20</y>
            <width>189</width>
            <height>23</height>
           </rect>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>255</x>
            <y>25</y>
            <width>46</width>
            <height>16</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>255</x>
            <y>45</y>
            <width>46</width>
            <height>16</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>245</y>
            <width>91</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>AutoCorrection</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>75</y>
            <width>86</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QwtCounter" name="Counter">
          <property name="geometry">
           <rect>
            <x>25</x>
            <y>220</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="numButtons">
           <number>3</number>
          </property>
         </widget>
         <widget class="QwtKnob" name="kNobDip">
          <property name="geometry">
           <rect>
            <x>14</x>
            <y>54</y>
            <width>211</width>
            <height>171</height>
           </rect>
          </property>
          <property name="upperBound">
           <double>1000.000000000000000</double>
          </property>
          <property name="knobStyle">
           <enum>QwtKnob::Sunken</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox02">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>276</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>276</height>
          </size>
         </property>
         <property name="title">
          <string>Color indicator</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>15</y>
            <width>55</width>
            <height>68</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="radioBRed">
             <property name="text">
              <string>Red</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioBGreen">
             <property name="text">
              <string>Green</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioBBlue">
             <property name="text">
              <string>&amp;Blue</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>278</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>158</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>158</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport_to_pdf"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Opt&amp;ions</string>
    </property>
    <addaction name="action_Settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp_BR"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Tools"/>
    <addaction name="actionRestore_Default_Layout"/>
    <addaction name="actionSave_Dafault_Layout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>752</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>55</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>&gt;&gt;&gt;&gt; &lt;&lt;&lt;&lt;</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>296</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="KLed" name="kled">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="state">
         <enum>KLed::Off</enum>
        </property>
        <property name="shape">
         <enum>KLed::Circular</enum>
        </property>
        <property name="look">
         <enum>KLed::Sunken</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="net_on_off">
        <property name="toolTip">
         <string comment="123" extracomment="321">Network Data</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Data</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonSet0">
        <property name="text">
         <string>&gt; 0 &lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonRestL">
        <property name="text">
         <string>&amp;Restore Layout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnExit">
        <property name="toolTip">
         <string>Exit</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/btn_exit.png</normalon>
         </iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionHelp_BR">
   <property name="text">
    <string>&amp;Help BR</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExport_to_pdf">
   <property name="text">
    <string>&amp;Export to pdf</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Tools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Tools</string>
   </property>
  </action>
  <action name="actionRestore_Default_Layout">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Restore Default Layout</string>
   </property>
   <property name="toolTip">
    <string>Restore Default Layout</string>
   </property>
  </action>
  <action name="actionSave_Dafault_Layout">
   <property name="text">
    <string>&amp;Save Dafault Layout</string>
   </property>
  </action>
  <zorder>dockWidget</zorder>
 </widget>
 <layoutdefault spacing="5" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
  <customwidget>
   <class>QwtCounter</class>
   <extends>QWidget</extends>
   <header>qwt_counter.h</header>
  </customwidget>
  <customwidget>
   <class>QwtKnob</class>
   <extends>QWidget</extends>
   <header>qwt_knob.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
  <include location="br.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Tools</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>net_on_off</sender>
   <signal>toggled(bool)</signal>
   <receiver>kled</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExit</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidget</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>action_Tools</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
